apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-kafka-0.11.x=org.apache.skywalking.apm.plugin.kafka.v1.define.CallbackInstrumentation
-kafka-0.11.x=org.apache.skywalking.apm.plugin.kafka.v1.define.KafkaConsumerInstrumentation
-kafka-0.11.x=org.apache.skywalking.apm.plugin.kafka.v1.define.KafkaProducerInstrumentation
-kafka-0.11.x=org.apache.skywalking.apm.plugin.kafka.v1.define.ProducerRecordInstrumentation
```

Comment:
Could you tell me why `ProducerRecordInstrumentation` is not required?

Reply:
From the existing codes, look like it set `dynamicField`, is this not required?

Reply:
TopicName can be obtained directly from ProducerRecord

Reply:
Got it, thanks.

Reply:
We can get the `topic` by `arguments` directly, so we don't need to enhance this.